crud/common/utils.lua

```diff
@@ -97,6 +97,12 @@ end
 
 function utils.get_space(space_name, replicasets)
     local replicaset = select(2, next(replicasets))
+
+    if replicaset.master.conn.error ~= nil then
+        local error_msg = string.format('The connection to the master is not valid: %s',
+                                         replicaset.master.conn.error)
+        error(error_msg)
+    end
```

Comment:
According to #95 it is also possible that `replicaset.master` is `nil`. The issue is pretty old, but maybe it worth to re-check it.

Reply:
Thanks! So far I've implemented a simple check `replicaset.master` is not `nil`, but it is not entirely clear to me where the timeout for this check is set from